app/javascript/components/workers-form/workers-form.jsx

```diff
@@ -0,0 +1,178 @@
+          reportingCount: selectCount(wb.queue_worker_base.reporting_worker.count),
+          webServiceMem: parseWorker(wb.web_service_worker).bytes || toBytes(memDefault),
+          webServiceCount: selectCount(wb.web_service_worker.count),
+          remoteCount: selectCount(wb.remote_console_worker.count),
+        };
+
+        setInitialValues({
+          ...parsedValues,
+        });
+
+        setSchema(() => addSchema(parsedValues));
+
+        setIsLoading(false);
+
+        miqSparkleOff();
+      })
+      .catch(
+        ({ error: { message } = { message: __('Could not fetch the data') } }) => {
+          add_flash(message, 'error');
+          miqSparkleOff();
+        },
+      );
+  }, []);
+
+  const onSubmit = (values) => {
+    // Compares initialValues and values and returns the value only if it's different
+    const isDifferent = name => (initialValues[name] !== values[name] ? values[name] : undefined);
+
+    miqSparkleOn();
+    const result = {
+      queue_worker_base: {
+        generic_worker: {
+          memory_threshold: toRubyMethod(isDifferent('genericMem')),
+          count: isDifferent('genericCount'),
+        },
+        priority_worker: {
+          memory_threshold: toRubyMethod(isDifferent('priorityMem')),
+          count: isDifferent('priorityCount'),
+        },
+        ems_metrics_processor_worker: {
+          memory_threshold: toRubyMethod(isDifferent('cuProcMem')),
+          count: isDifferent('cuProcCount'),
+        },
+        smart_proxy_worker: {
+          memory_threshold: toRubyMethod(isDifferent('analysisMem')),
+          count: isDifferent('analysisCount'),
+        },
+        ems_metrics_collector_worker: {
+          memory_threshold: toRubyMethod(isDifferent('cuColMem')),
+          count: isDifferent('cuColCount'),
+        },
+        ems_refresh_worker: {
+          memory_threshold: toRubyMethod(isDifferent('refreshMem')),
+        },
+        reporting_worker: {
+          memory_threshold: toRubyMethod(isDifferent('reportingMem')),
+          count: isDifferent('reportingCount'),
+        },
+      },
+      remote_console_worker: {
+        count: isDifferent('remoteCount'),
+      },
+      web_service_worker: {
+        memory_threshold: toRubyMethod(isDifferent('webServiceMem')),
+        count: isDifferent('webServiceCount'),
+      },
+      ui_worker: {
+        count: isDifferent('uiCount'),
+      },
+      vim_broker_worker: {
+        memory_threshold: toRubyMethod(isDifferent('brokerMem')),
+      },
+      event_catcher: {
+        memory_threshold: toRubyMethod(isDifferent('eventMonitorMem')),
+      },
+    };
+
+    const patch = {
+      workers: {
+        worker_base: {
+          ...buildPatch(result),
+        },
+      },
+    };
+
+    return API.patch(`/api/servers/${id}/settings`, patch)
+      .then(
+        () => {
+          add_flash(__(`Configuration settings saved for ${product} Server "${name} [${id}]" in Zone "${zone}"`), 'success');
```

Comment:
Translation wont work you need to use sprintf:
```
sprintf(__('%s:"%s" was successfully deleted'), toolbar.entity, entityName)
```

GetText does not get template strings